packages/block-library/src/editor.scss

```diff
@@ -25,7 +25,7 @@
 @import "./quote/editor.scss";
 @import "./rss/editor.scss";
 @import "./search/editor.scss";
-@import "./section/editor.scss";
+@import "./group/editor.scss";
```

Comment:
Can it be moved to be sorted alphabetically?

Reply:
👍 done